packages/collector/test/immediate/test.js

```diff
@@ -30,4 +34,59 @@ describe('collector/src/immediate', function() {
       expect(pidsThatContactedTheAgent[controls.getPid()]).to.exist;
     });
   });
+
+  describe('exclude processes from auto instrumentation', () => {
```

Comment:
This is the integration test for the change in `@instana/collector`. It starts an application under test via `npm start` and verifies that only the actual application under test is instrumented.